src/App.jsx

```diff
@@ -1,12 +1,151 @@
+            return 1;
+        }
+        if (a.population < b.population) {
+            return -1;
+        } else {
+            return 0;
+        }
+    })
+
+
+    async function fetchSingleCountry(event) {
+        event.preventDefault();
+        setError(false);
+
+        console.log(searchValue)
+
+        try {
+            const responseSingle = await axios.get(`https://restcountries.com/v3.1/name/${searchValue}`);
+            setChosenCountryInfo(responseSingle.data[0]);
+            //console.log(responseSingle.data[0].capital[0]);
+            //console.log(responseSingle.data[0].name.common);
+            setSearchValue('');
+        } catch (e) {
+            console.error(e);
+            setError2(true)
+        }
+    }
+
+
     return (
         <>
-            Maak je applicatie hier!
+            <div className="outer-container">
+                <div className="inner-container">
+
+                    <header>
+                        <img src={worldmap} alt="kaart van de wereld"/>
+
+                        <h1>World Regions</h1>
+                    </header>
+
+                    <main>
+
+                        <section className="country-function">
+                            {showButton &&
+                                <button type="button" className="country-button" onClick={fetchCountries}>Klik hier en
+                                    laat je verrassen </button>
+                            }
+                            {error &&
+                                <p className="error-message"> Er is iets mis gegaan met het ophalen van de gegevens.
+                                    Probeer het opnieuw </p>
+                            }
+
+                            <div className="card-container">
+
+                                {countries.map((country) => {
+
+                                    return (
+                                        <>
+                                            <div key={country.cca3} className="country-card">
+                                                <div className="wrapper">
+                                                    <img src={country.flags.png}
+                                                         alt={`Vlag van ${country.name.official}`}
+                                                         className="country-flag"/>
+
+                                                    <h3 className={`${continentColor(country)} country-name`}>{country.name.official}</h3>
+                                                </div>
+                                                <p>
+                                                    has a population of {country.population} people.
+                                                </p>
+                                            </div>
+                                        </>
+                                    );
+                                })}
+                            < /div>
```

Comment:
Semantisch gezien gaat het hier helemaal mis. Je hebt een div met daarin de map methode, die vervolgens een lege fragment retourneerd met daarna nog 2 divjes. Zo creeër je dus een tig aantal divjes en daarmee een divjes hel. 

Wanneer de map methode gebruikt binnenin de HTML kun je het beste daaromheen een ul element gebruiken. Vervolgens retourneer je na de map methode een li element waardoor je dus in semantisch opzicht het volgende hebt staan:

ul
  li
  li
  li
  etc..
ul

Dat is natuurlijk beter dan:

div
  div
     div
     div
     div
  div
div